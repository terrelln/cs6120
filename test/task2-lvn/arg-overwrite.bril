@main(a: int, b: int) {
    c: int = id a;
    d: int = id b;
    e: int = add a b;
    f: int = add a b;
    g: int = id e;
    a: int = id e;
    b: int = id f;
    h: int = add a b;
    i: int = add c d;
    j: int = add c d;
    k: int = id a;
    l: int = id b;
    print l;
}