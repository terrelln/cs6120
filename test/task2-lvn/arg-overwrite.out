@main(a: int, b: int) {
.__block0:
  e: int = add a b;
  a: int = id e;
  b: int = id e;
  print e;
}
